bin/compile

```diff
@@ -18,13 +18,19 @@ mkdir -p $CACHE_DIR $BIN_PATH $INSTALL_PATH
 if [[ -f "$ENV_DIR/WKHTMLTOPDF_VERSION" ]]; then
   WKHTMLTOPDF_VERSION=$(cat "$ENV_DIR/WKHTMLTOPDF_VERSION")
 else
-  WKHTMLTOPDF_VERSION="0.12.6-1"
+  if [[ "$STACK" = "scalingo-18" ]] || [[ "$STACK" = "scalingo-20" ]] ; then
+    WKHTMLTOPDF_VERSION="0.12.6-1"
+  elif [[ "$STACK" = "scalingo-22" ]] ; then
+    WKHTMLTOPDF_VERSION="0.12.6.1-2"
```

Comment:
Where does this version number comes from? Is it the latest version?

Reply:
scalingo 18 and 20 are not supported by the latest build of wkhtmltopdf:
Latests version supporting scalingo-22: https://github.com/wkhtmltopdf/packaging/releases/tag/0.12.6.1-2
Latests version supporting scalingo-18 and 20: https://github.com/wkhtmltopdf/packaging/releases/tag/0.12.6-1